### PR TITLE
Issue #301 - Updated additional resources for vertx. resolves #301

### LINKED ISSUE #301: Add additional Vert.X resources for background info in runtime guide

### PR DESCRIPTION
**Pull request by rhoads-zach**
@cescoffier @inoxx03 mind taking a look?

### CODE REVIEW

**Comment by cescoffier** [comment]
Looks good to me ! Thanks

File: docs/topics/vertx-dev-guide-additional-resources.adoc [hunk 1]
```diff
@@ -2,5 +2,10 @@
...
 * link:http://vertx.io[{VertX} project]
-* link:https://developers.redhat.com/promotions/building-reactive-microservices-in-java/[Building Reactive Microservices in Java book]
-* link:http://escoffier.me/vertx-hol/#_vert_x[Vert.x - From zero to (micro)-hero]
\ No newline at end of file
+* link:http://middlewareblog.redhat.com/2017/05/04/vert-x-for-reactive-programming-in-red-hat-openshift-application-runtimes/[{VertX} for Reactive Programming in Red Hat OpenShift Application Runtimes]
+* link:https://developers.redhat.com/promotions/building-reactive-microservices-in-java/[Building Reactive Microservices in Java]
+* link:https://developers.redhat.com/promotions/vertx-cheatsheet/[{VertX} Cheat Sheet for Developers]
+* link:http://escoffier.me/vertx-hol/#_vert_x[{VertX} - From zero to (micro)-hero]
+* link:https://github.com/cescoffier/rhsummit17-reactive-programming-with-vertx/blob/master/reactive-programming-with-vertx.pdf[Red Hat Summit 2017 Talk - Reactive Programming with {VertX}]
+* link:https://www.redhat.com/en/about/videos/reactive-systems-eclipse-vertx-and-red-hat-openshiftp[Red Hat Summit 2017 Breakout Session - Reactive Systems with {VertX} and Red Hat OpenShift]
```
Comment (inoxx03) [inline docs/topics/vertx-dev-guide-additional-resources.adoc]: This link does not work, I think there's an extra 'p' at the end of the URL
Should be 
`https://www.redhat.com/en/about/videos/reactive-systems-eclipse-vertx-and-red-hat-openshift`

Comment (rhoads-zach) [inline docs/topics/vertx-dev-guide-additional-resources.adoc]: Thanks @inoxx03 fixed.  merging now.

**Comment by rhoads-zach** [comment]
merged: https://github.com/openshiftio/appdev-documentation/commit/3deb1ad04b0e9e0b04700ba54140b97dd1bcfc2e

**Pull request closed without merging**